### DISCUSSION TITLE
When will we update node-openid-client to use latest `jose` version lib ?

### DISCUSSION THREAD

**Discussion by kdhttps** [panva/openid-client#383] [Q&A]
Currently `node-openid-client` uses old jose version `2.0.5`.

**Comment by panva** [comment]
With the next major release. There's nothing wrong with jose@2 as it is still [supported](https://github.com/panva/jose#supported-versions).

**Reply by kdhttps** [reply]
Agreed :+1: Thank you Sir :100:
